sigopt/orchestrate/kubectl/service.py

```diff
@@ -24,7 +25,8 @@ def __str__(self):
     )
 
 class KubectlService(Service):
-  kubectl_command = 'kubectl'
+  def kubectl_command(self):
```

Comment:
should it be `@property`?

Reply:
The others aren't. So I could make them properties too or just change to getters